docs/README.rst

```diff
@@ -244,19 +242,58 @@ distributed Python. You can think of it as a ``trio``
 stdlib's ``multiprocessing`` but built on async programming primitives
 from the ground up.
 
-``tractor``'s nurseries let you spawn ``trio`` *"actors"*: new Python
-processes which each run a ``trio`` scheduled runtime - a call to ``trio.run()``.
-
 Don't be scared off by this description. ``tractor`` **is just** ``trio``
 but with nurseries for process management and cancel-able streaming IPC.
 If you understand how to work with ``trio``, ``tractor`` will give you
-the parallelism you've been missing.
+the parallelism you may have been needing.
+
+
+Wait, huh?! I thought "actors" have messages, and mailboxes and stuff?!
+***********************************************************************
+Let's stop and ask how many canon actor model papers have you actually read?
+
+From the author's mouth, **the only thing required** is `adherance to`_
+the `3 axioms`_, *and that's it*.
+
+To get more fired up on the matter, please read these issues:
```

Comment:
This is minor, but I feel this section might read more fluidly if these "read more" lines were moved next to the `If you want do debate this further` paragraph, so that the reader goes through:
- Context (`From the author's mouth...`)
- "Flaws" in existing systems (`many "actor systems"...`)
- What ``tractor`` brings to the table (`tractor` aims to...)
- Further reading and discussion (`If you want do debate...)

What do you think?

Reply:
Yup sounds good.